GUI/QtQuick/TxListModel.cpp

```diff
@@ -242,50 +242,6 @@ void TxListModel::addRows(const std::vector<bs::TXEntry>& entries)
    }
 }
 
-void TxListModel::prependRow(const bs::TXEntry& entry)
```

Comment:
Make sure ZC TX is the first row in TX list (i.e. sorting is performed by nbConf)